src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ManagedGroupService.scala

```diff
@@ -19,12 +19,14 @@ class ManagedGroupService(private val resourceService: ResourceService, private
   def managedGroupType: ResourceType = resourceTypes.getOrElse(ManagedGroupService.managedGroupTypeName, throw new WorkbenchException(s"resource type ${ManagedGroupService.managedGroupTypeName.value} not found"))
   def memberRole = managedGroupType.roles.find(_.roleName == ManagedGroupService.memberRoleName).getOrElse(throw new WorkbenchException(s"${ManagedGroupService.memberRoleName} role does not exist in $managedGroupType"))
 
+  // TODO: What should be returned here?  Just a Resource object?  The report generated from the sync with Google?  Both?
   def createManagedGroup(groupId: ResourceId, userInfo: UserInfo): Future[Resource] = {
     for {
       managedGroup <- resourceService.createResource(managedGroupType, groupId, userInfo)
       _ <- createPolicyForMembers(managedGroup)
       policies <- accessPolicyDAO.listAccessPolicies(managedGroup)
-      _ <- createAggregateGroup(managedGroup, policies)
+      workbenchGroup <- createAggregateGroup(managedGroup, policies)
+      _ <- cloudExtensions.onGroupCreate(workbenchGroup.id)
```

Comment:
I think just call synchronizeGroupMembers directly

Reply:
Probably better to publish a message to the pub/sub topic instead of calling `synchronizeGroupMembers ` directly.

Reply:
I don't disagree, however, I believe implementing in this way would be non-trivial.  If anyone feels strongly that this logic should be written to use Google Pub/Sub, maybe that should be its own standalone issue?

Reply:
I think we have most of the code we need. See `GoogleExtensions.onGroupUpdate`. May just need to extract a method to keep the spirit of the `on[Event]` functions.

Reply:
Talked with @dvoet to help clarify how this works.  The existing `onGroupUpdate` functionality only updates Groups that already exist.  I added the `publishGroup` method to force the Group creation and use Google Pub/Sub.  I added the method to `CloudExtensions` so that we didn't have to pass a `GooglePubSubDAO` into the `ManagedGroupService` constructor.